config_converter/config_converter.py

```diff
@@ -0,0 +1,213 @@
+"""Program that converts a fluentd config file to a master agent config file.
+
+Usage: Run the exec file in the bin folder with 2 arguments - the
+path to the config file to migrate, and a directory to store the output
+master agent config file in.
+
+To do:
+1. Improve the CLI. Include a --help option. Allow optional args like
+   log-level and log-filepath (see project docs).
+2. Create a mapping for log_level if there in the fluentd config file and input
+   default ones if not given
+3. Generate stats like #fields converted, ignored, and output as schema
+4. Generate logs
+5. Fix types of numbers, lists
+6. Generate tests to check if for right logs, right stats, above changes
+"""
+
+import copy
+import sys
+import os
+import subprocess
+import yaml
+from google.protobuf import json_format
+import config_converter.config_pb2 as config_pb2
+
+
+class ConfigConverter():
+    """Class containing tools to map fluentd fields to master agent fields."""
+    def __init__(self, config_obj):
+        self.result = {'logs_module': dict()}
+        self._create_mapping_info()
+        self.config_obj = config_obj
```

Comment:
I think `config_obj` is only used by _extract_root_dirs(), so it can just be an input parameter of the function `_extract_root_dirs()`, and we don't need to store config_obj as a member data.

Reply:
Sounds good!